### DISCUSSION TITLE
ability to choose camera source

### DISCUSSION THREAD

**Discussion by Saganaki22** [hacksider/Deep-Live-Cam#100] [Ideas]
it would be useful to add choice of cam source [1-4] for example as some people have more than 1 camera

**Comment by Saganaki22** [comment]
this fork has done this already https://github.com/samoylenkodmitry/Deep-Live-Cam

**Comment by hacksider** [comment]
This is already on the experimental build. ☺️

**Reply by Saganaki22** [reply]
this was back when it wasnt thanks for your reply tho

**Reply by hacksider** [reply]
Yes, thanks for mentioning that. I also just replied to the thread to ensure this was also answered by me ☺️

**Reply by Saganaki22** [reply]
thank you
